beacons/src/commonTest/kotlin/BeaconUpdateTest.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+import BeaconMock.mockBeaconDevice
```

Comment:
can just normally namespace this.

Reply:
Do mean like this: `com.splendo.kaluga.bluetooth.beacons.BeaconMock.mockBeaconDevice`?

Reply:
Got it, just use `BeaconMock.mockBeaconDevice`

Reply:
yeah it should just be in the right package, then no import is needed.